### PR TITLE
feat: ms-catalog specific doc changes

### PR DESCRIPTION
**Pull request by NeethuES-intel**
## PR Checklist

- [X] Added label to the Pull Request for easier discoverability and search
- [X] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
Changes are specific to MS-catalog, as referenced here - https://github.com/intel-innersource/applications.services.esh.intel-edge-services-document

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

**Review by antoniomtz — APPROVED** [review]

File: .github/workflows/builddocumentation.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      branches: [ 'main', 'master' ]
+      types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
+      paths:
+      - 'docs/**'
+env:
+    http_proxy: http://proxy-chain.intel.com:912
```
Comment (jim-wang-intel) [inline .github/workflows/builddocumentation.yml:14]:
```suggestion
    http_proxy: http://proxy-chain.intel.com:911
```

File: .github/workflows/publishtoaws.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+   name: PublishToAWS
+   on:
+    workflow_dispatch:
+   env:
+      # proxy-dmz mangles DNS resolution, resulting in certificate verification failure
+      http_proxy: http://proxy-chain.intel.com:912
```
Comment (jim-wang-intel) [inline .github/workflows/publishtoaws.yml:6]:
```suggestion
      http_proxy: http://proxy-chain.intel.com:911
```

**Review by jim-wang-intel — COMMENTED** [review]
not sure if it matter for http_proxy setting or not but i think the port for that is 911 unless there is some change that i am not aware recently.

File: .github/workflows/builddocumentation.yml (hunk 1 above)
Comment (NeethuES-intel) [reply .github/workflows/builddocumentation.yml:14]: this file is copy paste from their examples.

Comment (jim-wang-intel) [reply .github/workflows/builddocumentation.yml:14]: https://intelpedia.intel.com/Intel_Proxy_servers

Comment (NeethuES-intel) [reply .github/workflows/builddocumentation.yml:14]: Link says 911 for http & 912 for https

Comment (jim-wang-intel) [reply .github/workflows/builddocumentation.yml:14]: yes, so port for **http_proxy** is 911 and port for **https_proxy** is 912

File: src/Dockerfile [hunk 3]
```diff
@@ -8,12 +8,12 @@ FROM intel/dlstreamer:2023.0.0-ubuntu22-gpu682-dpcpp as base
 USER root
 
 FROM base as build-default
-COPY ./requirements.txt /requirements.txt
+COPY src/requirements.txt /requirements.txt
```
Comment (brian-intel) [inline src/Dockerfile:11]: For this to work you'll need to modify the make file build command. Currently it uses "src/" for the build context when it should now use "."

https://github.com/NeethuES-intel/automated-self-checkout/blob/ms-catalog/Makefile#L34
https://github.com/NeethuES-intel/automated-self-checkout/blob/ms-catalog/Makefile#L37

**Review by brian-intel — CHANGES_REQUESTED** [review]

**Comment by NeethuES-intel** [comment]
Closing PR, as Edge Developer Toolbox doesn't support documentation + package creation for intel open source repos.

**Pull request closed without merging**